core/lib/dal/src/blocks_dal.rs

```diff
@@ -1116,6 +1116,7 @@ impl BlocksDal<'_, '_> {
         storage_refunds: &[u32],
         pubdata_costs: &[i32],
         predicted_circuits_by_type: CircuitStatistic, // predicted number of circuits for each circuit type
+        bytes_per_blob: u64,
```

Comment:
this is always a constant, right?

Reply:
yes, but it's coming from kzg crate, so we don't want dal to depend on it and we're passing it separately